.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Continous Integration
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+
+env:
+  CI: true
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        node-version: [12, 14, 16]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: ankane/setup-mysql@v1
+        with:
+          mysql-version: 5.7
+      - run: |
+          sudo mysql -e "CREATE USER 'test'@'localhost' IDENTIFIED BY 'test'"
+          sudo mysql -u root -e "GRANT ALL PRIVILEGES ON *.* TO 'test'@'localhost'"
+          sudo mysql -u root -e "ALTER USER 'test'@'localhost' IDENTIFIED WITH mysql_native_password by 'test'"
+          sudo mysql -u root -e "FLUSH PRIVILEGES"
+      - run: npm install
+      - run: npm test
+        env:
+          MYSQL_USER: test
```

Comment:
You can use gitHub Secrets {{ }} here instead of the test/test.

Reply:
let me set it up. thanks for the suggestion.